Makefile

```diff
@@ -206,6 +215,10 @@ eunit-roma: KIND=test
 eunit-roma: internal-build
 	@ERL_FLAGS="-args_file $(EUNIT_VM_ARGS) -config $(EUNIT_SYS_CONFIG) -network_id local_roma_testnet" ./rebar3 do eunit $(EUNIT_TEST_FLAGS)
 
+eunit-fortuna: KIND=test
```

Comment:
Circle CI config?

Reply:
I wasn't sure we wanted to enable it immediately, the tests would be identical to the minerva tests until we actually start using it... But probably better to activate immediately.